src/components/AuthorProfile.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Avatar from '@mui/material/Avatar';
+import Container from '@mui/material/Container';
+// import Skeleton from '@mui/material/Skeleton';
+import { UpskillClubApi } from '../apis';
+import { Utils } from '../common';
+import { Author, GetAuthorByIdResponse } from './interface';
+
+const renderAuthorProfileLoading = () => {
+  return (
+    <Container>
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+        <Typography variant="h3" marginBottom="5px">
+          Loading...
+        </Typography>
+      </Box>
+    </Container>
+  );
+};
+
+const AuthorProfile = () => {
+  const { id } = useParams();
+
+  const [authorProfileData, setAuthorProfileData] = React.useState<Author>();
+  const [authorProfileLoading, setAuthorProfileLoading] = React.useState(false);
+  const [authorProfilePresent, setAuthorProfilePresent] = React.useState(true);
```

Comment:
Fetch sessions by the author also.
`curl --location --request GET 'https://sourabhjaz.pythonanywhere.com//api/session?author=3'`

Reply:
done